src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -3255,6 +3256,14 @@ void GlobalObject::finishCreation(VM& vm)
                     ProcessBindingConstants::createStructure(init.vm, init.owner)));
         });
 
+    m_processBindingFs.initLater(
```

Comment:
needs visit call in `void GlobalObject::visitChildrenImpl(JSCell* cell, Visitor& visitor)`